WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -38,8 +38,10 @@ + (NSArray *)routes
     [[FBRoute POST:@"/wda/homescreen"].withoutSession respondWithTarget:self action:@selector(handleHomescreenCommand:)],
     [[FBRoute POST:@"/wda/deactivateApp"] respondWithTarget:self action:@selector(handleDeactivateAppCommand:)],
     [[FBRoute POST:@"/wda/keyboard/dismiss"] respondWithTarget:self action:@selector(handleDismissKeyboardCommand:)],
-    [[FBRoute GET:@"/lock"].withoutSession respondWithTarget:self action:@selector(handleLock:)]
-  ];
+    [[FBRoute PUT:@"/lock"].withoutSession respondWithTarget:self action:@selector(handleLock:)],
```

Comment:
do we need PUT endpoints? I'd rather create to POST for changing the state and one for getting the current state

Reply:
laso for Appium it makes sense to duplicate these endpoints, so they can be also used together with session id